fix(curriculum): update fcc author wording (#60050)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-fcc-authors-page/641da5abaac81844a54adb03.md

Now, it is time to create the function that will be responsible for displaying the authors on the page. 

> Start by creating a function called `displayAuthors` that takes an `authors` parameter. Your function should then loop through the `authors` array and add a card for each author to the `div` with an `id` of `authorContainer`. 

Reviewer: ```suggestion
Start by creating a function called `displayAuthors` that takes an `authors` parameter. Your function should then loop through the `authors` array and add a card for each author to the `#authorContainer` element. 
```

I would make it more concise
